made utilities and experiments compatible

debobiswas01 opened the pull request:
following changes in utilities folder: 
added imports 
also uses the save_data defined from utilities now

also made all experiments compatible with utilities

debobiswas01 committed:
made utilities and experiments compatible

comp-phys-marc commented:
Let's just try and see if these run against the MultiVu in simulation mode please.

debobiswas01 committed:
changes to utilities

added the . in __init__ file to import modules correctly

changed the timing by sampling more number of points (math.ceil) and decreased wait time by changing points to (points-1)

debobiswas01 committed:
second commit to remove the log and resistance data files

comp-phys-marc commented:
Please also remove the __pycahce__.

comp-phys-marc reviewed (changes requested):
Please remove the __pycahce__

debobiswas01 committed:
removed_stash_response_to_feedback

comp-phys-marc reviewed (changes requested):
One last thing, please add the __pycache__ to the .gitignore so that it does not get generated and added automatically again.
